preview/scripts/edit-description.js

```diff
@@ -9,7 +9,12 @@ module.exports = async ({inputs, github, context}) => {
     if (RTD_PLATFORM == "business") {
         var RTD_DOMAIN = "com.readthedocs.build";
     }
-    var RTD_URL = `https://${RTD_PROJECT_SLUG}--${PR_NUMBER}.${RTD_DOMAIN}/${RTD_PROJECT_LANGUAGE}/${PR_NUMBER}/`;
+    var RTD_DOMAIN = `https://${RTD_PROJECT_SLUG}--${PR_NUMBER}.${RTD_DOMAIN}/`;
+    if (RTD_SINGLE_VERSION == "true") {
```

Comment:
There is almost certainly a better way to do this, but it seemed most explicit?

Reply:
Why not using a boolean instead?

```suggestion
    if (RTD_SINGLE_VERSION) {
```